vspace/src/main/java/edu/asu/diging/vspace/core/model/display/impl/ExternalLinkDisplay.java

```diff
@@ -0,0 +1,72 @@
+package edu.asu.diging.vspace.core.model.display.impl;
+
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.OneToOne;
+
+import org.hibernate.annotations.GenericGenerator;
+import org.hibernate.annotations.Parameter;
+
+import edu.asu.diging.vspace.core.model.IExternalLink;
+import edu.asu.diging.vspace.core.model.display.IExternalLinkDisplay;
+import edu.asu.diging.vspace.core.model.impl.ExternalLink;
+
+import javax.persistence.Entity;
+
+@Entity
+public class ExternalLinkDisplay extends LinkDisplay implements IExternalLinkDisplay {
+
+    @Id
+    @GeneratedValue(generator = "link-display-id-generator")
+    @GenericGenerator(name = "externallink-display-id-generator", 
+      parameters = @Parameter(name = "prefix", value = "EXLD000000000"),
```

Comment:
why has the prefix all the 0s? this should be part of the number formatting, not the prefix.